scripts/deploy.sh

```diff
@@ -239,6 +254,13 @@ _install_pip_pkgs_clr() {
   sudo bash -c "https_proxy=$https_proxy source venv/bin/activate && https_proxy=$https_proxy pip3 --log $log install --no-binary psycopg2 -r $reqs"
 }
 
+_install_pip_pkgs_centos() {
```

Comment:
(This is petty so disregard if you don't agree)

_install_pip_pkgs_clr (line #234) and .._ubuntu (line #227) were created because the binaries for psycopg2 don't play well with Clear Linux.  Consider refactoring _install_pip_pkgs_ubuntu and _install_pip_pkgs_centos to call a new function "_install_pip_pckgs_" with the content that _install_pip_pkgs_ubuntu has right now.  This is because having two blocks of code which bodies look exactly the same should be avoided when possible.

Reply:
I agree

Reply:
+1 to this change. The *_${DISTRO} functions are nice to use when the logic between distros is sufficiently different to warrant the separation.